docs/GettingStartedDocs/IntelSDKPortingGuideLinux.md

```diff
@@ -0,0 +1,306 @@
+# Intel SGX SDK to Open Enclave SDK Porting Guide (Linux)
```

Comment:
Since the project was submitted to the CCC, the name has been listed in CCC docs as "SGX SDK for Linux" which is how it appears at http://confidentialcomputing.io (click on "Projects"), rather than "Intel SGX SDK".   Clicking on that link takes you to an Intel page that still calls it the "Intel SGX SDK for Linux" (but notice it still has "for Linux" in the SDK name).

Since it hasn't completed the CCC acceptance process yet, it's up to you whether you prefix it with "Intel" (which implies you want to advertise that it's NOT yet a full CCC project) or remove the "Intel" (which implies it IS a CCC project like the CCC home page implies), but all references in this doc to the name should be consistent in terms of keeping the "... for Linux" suffix in the SDK name.